JRIngram/maze-gen
# Release 2.0.0

# Pull Request (PR) - *Release 2.0.0*
## Description of Change
### 2.0.0
#### Added
* Ability to use a seeded random number generator to generate a maze.
* Feature request template.
* .npmignore.
* Hunt and kill algorithm.
* Created a CONTRIBUTING.md file

#### Changes
* String representation of the maze to be more clear
* Bump jest from 25.1.0 to 25.4.0
* Improved bug report template.
* Order of parameters for generateMaze.
* Tidied tests and remove unused functions.
* Restructured README:
    * Added links to changelog and contributing
    * Added algorithm descriptions
* Moved linting and unit tests to distinct jobs in circleci


## Link To Issue 



## How to test this change
Run unit tests and linter.
Install local version of the package and test that the package runs as expected.


## Checklists
### Checklist for developer of feature

### Checklist for reviewers
